cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -307,13 +310,20 @@ class Instruction extends Construction::TInstruction {
     result = getResultString() + " = " + getOperationString() + " " + getOperandsString()
   }
 
+  predicate shouldGenerateDumpStrings() {
```

Comment:
```suggestion
  private predicate shouldGenerateDumpStrings() {
```

Reply:
Oops, missed this before I merged. I'll fix in another PR immediately.